src/app/pages/TopicPage/Curation/CurationGrid/index.jsx

```diff
@@ -27,6 +27,21 @@ const TopicList = styled.ul`
 `;
 
 const Item = styled.div`
+.promo-image {
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    width: 33%;
+    display: inline-block;
+    vertical-align: top;
+  }
+}
+.promo-text {
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    width: 67%;
+    display: inline-block;
+    vertical-align: top;
+    padding-inline-start: ${GEL_SPACING};
+  }
+}
```

Comment:
Similar here, where possible I think we should be using the Theme values for any new CSS styling

Reply:
@amoore108 Going to change this file to use the css prop in a separate PR